python/psautohint/__init__.py

```diff
@@ -69,3 +70,14 @@ def hint_compatible_bez_glyphs(info, glyphs, masters):
                                     tuple(tobytes(m) for m in masters))
 
     return [tounicode(g) for g in hinted]
+
+
+AUTOHINTEXE = os.path.join(
+    os.path.dirname(__file__),
+    "autohintexe" + (".exe" if sys.platform in ("win32", "cygwin") else "")
+)
+if not os.path.isfile(AUTOHINTEXE) or not os.access(AUTOHINTEXE, os.X_OK):
+    import warnings
+    warnings.warn(
+        "embedded 'autohintexe' executable not found: %r" % AUTOHINTEXE
+    )
```

Comment:
Lets not bother with this here and just provide the binary. AFDKO’s `autohint.py` can have all this magic.

Reply:
Keepin it doesn’t hurt though